t/config.t

```diff
@@ -157,6 +162,28 @@ subtest 'Test configuration override from file' => sub {
         ],
         'referers parsed correctly'
     );
+    my $untracked_assets_patterns = $app->config->{'assets/storage_duration'};
+
+    ok(ref $untracked_assets_patterns eq ref {}, "failed to read config for assets/storage_duration");
+    ok(scalar %$untracked_assets_patterns, "config is empty for assets/storage_duration");
+
+    return unless (ref $untracked_assets_patterns eq ref {} and scalar %$untracked_assets_patterns);
+
+    my $current_is_here    = 0;
+    my $current_value      = 0;
+    my $asset_name_matches = 0;
+    for my $pattern (keys %$untracked_assets_patterns) {
+        if ($pattern eq 'CURRENT') {
+            $current_is_here = 1;
+            $current_value   = $untracked_assets_patterns->{$pattern};
+            if ('SLE-12-SP5-HA-POOL-x86_64-CURRENT-Media1' =~ $pattern) {
+                $asset_name_matches = 1;
+            }
+        }
+    }
+    ok($current_is_here,     'CURRENT is configured');
```

Comment:
I think it would look nicer to just use `grep` instead of a for-loop for this check

Reply:
Agreed, this is way more verbose than it has to be.